jdtls.ext/com.microsoft.java.maven.plugin/plugin.xml

```diff
@@ -4,7 +4,8 @@
   <extension point="org.eclipse.jdt.ls.core.delegateCommandHandler">
     <delegateCommandHandler class="com.microsoft.java.maven.handler.DelegateCommandHandler">
       <command id="java.maven.initializeSearcher"/>
-      <command id="java.maven.searchArtifact"/>
+      <command id="java.maven.searchArtifactByClassName"/>
```

Comment:
consider to use a single entry `java.maven.searchArtifact`. 
Define the structure of the args, e.g. 
```ts
SearchArtifactArgument {
  searchType: enum;
  classname?: string;
  groupId?: string;
  artifactId?: string;
  ...
}
```

Reply:
ok.